src/config.rs

```diff
@@ -57,10 +57,12 @@ pub struct FederationConfig<T: Clone> {
     pub(crate) client: ClientWithMiddleware,
     /// Number of tasks that can be in-flight concurrently.
     /// Tasks are retried once after a minute, then put into the retry queue
+    /// Setting this count to `0` means that there is no limit
     #[builder(default = "1024")]
     pub(crate) worker_count: usize,
     /// Number of concurrent tasks that are being retried in-flight concurrently.
     /// Tasks are retried after an hour, then again in 60 hours.
+    /// Setting this count to `0` means that there is no limit
     #[builder(default = "128")]
     pub(crate) retry_count: usize,
```

Comment:
I would prefer to default these to 0 (unlimited). That way its easy to point someone to these settings in case of excessive memory use. On the other hand, it will lead to a lot of confusion if federation stops working reliably for no obvious reason once an instance gets too big.

Reply:
Done